extensions/grpc/codegen/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>quarkus-grpc-parent</artifactId>
+    <groupId>io.quarkus</groupId>
+    <version>999-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>quarkus-grpc-codegen</artifactId>
+  <name>Quarkus - gRPC - Code Gen</name>
```

Comment:
I would prefer if we could stick to the four spaces convention.

It's not consistent across all the source code but most of our pom files are this way.

Reply:
Is it possible to add this to our formatter?

Reply:
fixed